src/reader.cpp

```diff
@@ -261,6 +261,7 @@ class reader_data_t {
         : allow_autosuggestion(0),
           suppress_autosuggestion(0),
           expand_abbreviations(0),
+          silent(0),
```

Comment:
s/0/false/ please and you might as well fix the other bool initializers while you're touching this block of code :smile: Functionally it doesn't matter but it does make the intent clearer to a human reader.